### ISSUE TITLE
Add a fork me on Github logo

### ISSUE THREAD

**Issue by sara-02** [RedHatOfficial/RedHatOfficial.github.io#174]
From the website, there is no way to find the GitHub repo associated with the website. It would be great if we could add a `fork me on GitHub logo` to it. 
This will increase the visibility of the project, and help external users to pitch in their queries, add issues, and/or give suggestions about content and design.
Otherwise one has to perform an explicit search to find the Github repo associated with the project.

**Comment by webknjaz** [comment]
:+1: it took me a while to get here from the web site as well...

**Comment by castastrophe** [comment]
Great suggestion!  Would definitely accept pull requests with said update.

**Comment by Nachichuri** [comment]
Hey! I noticed this issue was still open so I added a button in #270, let me know what you think about it 😀 I guess it's a WIP since I couldn't try it in all the required browsers, so it'd be great if you could test it with the [netlify deploy preview](https://deploy-preview-270--red-hat.netlify.app/#!/main)!

**Comment by castastrophe** [comment]
I love the idea of linking to our GitHub Organization: https://github.com/RedHatOfficial, rather than a link directly to this project. Alternatively, maybe a link in/near the footer to open an issue?

**Comment by bproffitt** [comment]
I agree; there are a lot of cool projects happening in the organization at large. Can we have the GitHub button do this when it's added? (I can't seem to see it either.)

**Comment by Aanchal116** [comment]
Hii @castastrophe , the idea seems interesting. May I work on this issue?

**Comment by EraKin575** [comment]
Hi @castastrophe .I would like to work on this issue
